admin/partials/nginx-helper-admin-display.php

```diff
@@ -35,28 +35,35 @@
 					echo wp_kses(
 						sprintf(
 							'<a class="nav-tab%1$s" href="?page=nginx&tab=%2$s">%3$s</a>',
-							esc_attr( $class ), esc_attr( $setting_name['menu_slug'] ), esc_html( $setting_name['menu_title'] )
```

Comment:
Same as previous use case of `sprintf`. 

Could we concatenate `$setting_name['menu_slug'] ` with static value in `href` and pass it to `esc_url` instead?